frontend/app/utils/env.server.ts

```diff
@@ -178,13 +179,18 @@ export function getPublicEnv() {
 
   return result.data;
 }
+const log = getLogger('env.server');
```

Comment:
We usually put this at the top of the file (after the imports) so it can be used by any/all functions.

Reply:
Also makes sense. Implemented in commit 6de167b3b81f6af6f5d21ba1c58a95794b273d32.